managers/oclcClassify.py

```diff
@@ -43,7 +43,7 @@ class ClassifyManager:
     ]
 
     # This is a compressed 917kb file that contains a language detection model
-    LANG_MODEL = fasttext.load_model('lid.176.ftz')
+    LANG_MODEL = Word2Vec.load('lid.176.ftz')
```

Comment:
From the tests it seems like gensim can't read this file, not sure if there's another way to load it or if we need to take a different approach